pkg/collector/corechecks/embed/jmx/check.go

```diff
@@ -62,68 +62,69 @@ func (c *JMXCheck) Run() error {
 	return nil
 }
 
-// Stop TODO <agent-core> : IML-199
+// Stop attempts to force the JMXCheck to stop and will unschedule it
```

Comment:
```suggestion
// Stop forces the JMXCheck to stop and will unschedule it
```
This function doesn't return an error, so I don't know if "attempts" is a good word, no way to signal failure.